### ISSUE TITLE
Metadata parsing error

### ISSUE THREAD

**Issue by MeFisto94** [MaulingMonkey/cargo-vs#5]
When trying to invoke `cargo vs2022` on my project, I get
```
error parsing `cargo metadata`: unable to parse .cargo\git\checkouts\wow_messages-a636457bd164f1ad\98a7745\wow_login_messages\Cargo.toml: invalid type: map, expected a string for key `package.repository` at line 
11 column 24
```

Here's why: https://github.com/gtker/wow_messages/blob/main/wow_login_messages/Cargo.toml#L11
I had included the crate as such: `wow_login_messages = { git = "https://github.com/gtker/wow_messages", features = ["sync"] }`

**Comment by MaulingMonkey** [comment]
Should be fixed in the newly published 0.1.2, thanks for the bug report!
```cmd
cargo install cargo-vs
```

**Comment by MeFisto94** [comment]
Confirmed, works! Thank you!
